internal/wireguard/config.go

```diff
@@ -42,9 +43,11 @@ func makeDeviceConfig(settings Settings) (config wgtypes.Config, err error) {
 		}
 		preSharedKey = &preSharedKeyValue
 	}
-
+	var persistenKeepAlive time.Duration
+	if settings.PersistenKeepAlive > 0 {
+		persistenKeepAlive = time.Second * time.Duration(settings.PersistenKeepAlive)
+	}
```

Comment:
With settings.PersistentKeepAlive as time.Duration directly, no need to do conversion etc here.